docs/inventory_plugins.md

```diff
@@ -1,153 +1,79 @@
-tracking will still be maintained via the host `instance_id`.
-(after: https://github.com/ansible/awx/pull/3362)
+In many cases, the host names will change. In all cases, accurate host tracking will still be maintained via the host `instance_id`. (after: https://github.com/ansible/awx/pull/3362)
 
-## How do I write my own Inventory File?
+## Writing Your Own Inventory File
 
-If you do not want any of this compatibility-related functionality, then
-you can add an SCM inventory source that points to your own file.
-You can also apply a credential of a `managed_by_tower` type to that inventory
-source that matches the credential you are using, as long as that is
-not `gce` or `openstack`.
+If you do not want any of this compatibility-related functionality, then you can add an SCM inventory source that points to your own file. You can also apply a credential of a `managed_by_tower` type to that inventory source that matches the credential you are using, as long as it is not `gce` or `openstack`.
 
-All other sources provide _secrets_ via environment variables, so this
-can be re-used without any problems for SCM-based inventory, and your
-inventory file can be used securely to specify non-sensitive configuration
-details such as the keyed_groups to provide, or hostvars to construct.
+All other sources provide _secrets_ via environment variables.  These can be re-used without any problems for SCM-based inventory, and your inventory file can be used securely to specify non-sensitive configuration details such as the `keyed_groups` (to provide) or hostvars (to construct).
 
 ## Notes on Technical Implementation of Injectors
 
-For an inventory source with a given value of the `source` field that is
-of the built-in sources, a credential of the corresponding
-credential type is required in most cases (exception being ec2 IAM roles).
-This privileged credential is obtained by the method `get_cloud_credential`.
+For an inventory source with a given value of the `source` field that is of the built-in sources, a credential of the corresponding credential type is required in most cases (ec2 IAM roles are an exception). This privileged credential is obtained by the method `get_cloud_credential`.
 
-The `inputs` for this credential constitute one source of data for running
-inventory updates. The following fields from the
-`InventoryUpdate` model are also data sources, including:
+The `inputs` for this credential constitute one source of data for running inventory updates. The following fields from the `InventoryUpdate` model are also data sources, including:
 
  - `source_vars`
  - `source_regions`
  - `instance_filters`
  - `group_by`
 
-The way these data are applied to the environment (including files and 
-environment vars) is highly dependent on the specific source.
+The way these data are applied to the environment (including files and environment vars) is highly dependent on the specific source.
```

Comment:
"The way this data is applied"  sounds much better to me, but maybe there is some grammar technicality I don't know about here..

Reply:
Your suggestion sounds a lot better, editing this.